pgml-dashboard/content/.ignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
Why are you ignoring everything here?

Reply:
This is the `.ignore` for cargo-watch, not for git.